genericjpa/db/src/main/java/org/hibernate/search/annotations/UpdateInfo.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.annotations;
+
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Martin Braun
+ * @hsearch.experimental
+ */
+@Target({FIELD, METHOD, TYPE})
+@Retention(RUNTIME)
+@Repeatable(UpdateInfos.class)
+public @interface UpdateInfo {
```

Comment:
What's the meaning of this? Btw. I'd prefer another package name than "annotations". I wouldn't package by technical aspects (you don't want a package for all your enums either for instance). Rather something more meaningful like "async" or whatever this _semantically_ represents. Also the split package gives bad vibes for OSGi.